DataFormats/Common/src/classes_def.xml

```diff
@@ -204,4 +204,8 @@
   <version ClassVersion="3" checksum="2237762164"/>
  </class>
  <class name="edm::Wrapper<std::map<edm::ParameterSetID,edm::ParameterSetBlob>>"/>
+
+ <class name="edm::StdArray<short, 4>" ClassVersion="3">
```

Comment:
Hi @fwyzard sorry I realized this only now. For instantiations of class templates we don't express the class versions this way. Instead, the `edm::StdArray` class template definition should define its own version along
```cpp
#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"

namespace edm {
  template <typename T, std::size_t N>
  class StdArray {
  public:
    ...
    //Used by ROOT storage
    CMS_CLASS_VERSION(3);
    ...
  };
}
```
The macro adds a static function, so it shouldn't impact on the usability on GPUs.

Reply:
Ah... so I should add `CMS_CLASS_VERSION` to the header file, and change the XML to just `<class name="edm::StdArray<short, 4>"/>` ?

Reply:
> Ah... so I should add `CMS_CLASS_VERSION` to the header file, and change the XML to just `<class name="edm::StdArray<short, 4>"/>` ?

Correct. Thanks!

Reply:
Done.